android/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
+    buildToolsVersion "26.0.1"
```

Comment:
Do you think there's a way to do this conditionally? So projects not using RN 56 do not have to update their SDK versions?

Reply:
Check new Commit. I've Fixed this.